index.js

```diff
@@ -1,6 +1,7 @@
 import { getIndexProviderPeerIdFromSmartContract } from './lib/smart-contract-client.js'
 import { getIndexProviderPeerIdFromFilecoinMinerInfo } from './lib/filecoin-rpc-client.js'
 import { rpc } from './lib/rpc-client.js'
+import assert from 'assert'
```

Comment:
```suggestion
import assert from 'node:assert'
```

For consistency with the rest of our codebase.